core/src/datayoga_core/prom.py

```diff
@@ -0,0 +1,11 @@
+from prometheus_client import Counter, start_http_server
+
+incoming_records = Counter('incoming_records', 'Number of incoming records')
+
```

Comment:
any particular reason to have the separated? does this imply on different metric groups?

Reply:
✅